### PR TITLE
ome-files: Wrapper documentation fixes

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
- Remove last two remaining instances of `ome-files-test`; now called `ome-files`
- Document that `ome-files view` may not work if support was not compiled in

### CODE REVIEW

File: cmake/TemplateShellWrapper.cmake.in [hunk 1]
```diff
@@ -161,7 +161,7 @@ run() {
   realcmd="$LIBEXECDIR/$realcmd"
 
   if [ ! -x "$realcmd" ]; then
-    echo "ome-files: '$runcmd' is not a ome-files-test command.  See 'ome-files-test --help'" >&2
+    echo "ome-files: '$runcmd' is not a ome-files command.  See 'ome-files-test --help'" >&2
```
Comment (mtbc) [inline cmake/TemplateShellWrapper.cmake.in]: `ome-files-test` still needs fixing.

Comment (rleigh-codelibre) [inline cmake/TemplateShellWrapper.cmake.in]: Argh!

**Comment by mtbc** [comment]
Manpage now has,

```
       view (or glview)
              View image pixel data [optional component; only present if built
              with Qt5 and OpenGL support]
```

File: cmake/TemplateShellWrapper.cmake.in (hunk 1 above)
Comment (rleigh-codelibre) [inline cmake/TemplateShellWrapper.cmake.in]: Fix pushed.

**Comment by mtbc** [comment]
```shell
$ ome-files-standalone-bundle-0.4.0-DEV-59850d7-Debug-Ubuntu14.04-x86_64-b642/bin/ome-files view
ome-files: 'view' is not a ome-files command.  See 'ome-files --help'
```

**Pull request merged**